swift/ql/test/extractor-tests/declarations/declarations.swift

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
whoops, this newline is unintended, and changes the test results too much, I will revert this